projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/index.tsx

```diff
@@ -33,10 +33,12 @@ const VideopressCard: ProductCardComponent = props => {
 		videoCount,
 	} );
 
-	const customLoadTracks = {
-		stats_period: featuredStats?.period,
-		video_count: videoCount,
-	};
+	const customLoadTracks = useMemo( () => {
```

Comment:
Without being memoized, this was causing the videopress card to be unnecessarily re-rendered, which also caused multiple tracks to fire for its loading event